packages/api/lambdas/cleanExecutions.js

```diff
@@ -91,7 +92,10 @@ const cleanupExpiredESExecutionPayloads = async (
   let taskStatus;
   // this async and poll method allows us to avoid http timeouts
   // and persist in case of lambda timeout
-  log.info(`launched async ES task id ${updateTask.body.task}`);
+  log.info(`launched async ES task id ${updateTask.body.task}
```

Comment:
Let's spell out "ElasticSearch" to be painfully clear

Reply:
Was the suggestion to log these instructions or to return them in the Lambda response?

Reply:
my thinkingg was that these should be logged, because the us case is if lambda has not exited normally (and so we probably haven't gotten it's response)